src/styles/themes/ThemeProvider.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import ThemeContext from './ThemeContext';
+import useThemePreference from './useThemePreference';
+
+// Going to eventually import the light theme here too
+import darkTheme from './default';
+
+// Temporarily add a light theme here
+const lightTheme = {
+    appBG: '#F9F4F0',
+    text: '#001F40',
+};
```

Comment:
Was this for QA? Once we're ready to merge, copy into body of description and remove?